types/is-date-object/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for is-date-object 1.0
+// Project: https://github.com/ljharb/is-date-object
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+declare function isDateObject(value: Date): true;
+declare function isDateObject(value?: object): boolean;
+declare function isDateObject(value?: null | boolean | string | number): false;
```

Comment:
i assume TS 3.0 had arrays?
```suggestion
declare function isDateObject(value?: null | boolean | string | number | []): false;
```